examples/step-61/step-61.cc

```diff
@@ -0,0 +1,885 @@
+           // This part is the same as the part in evaluating pressure.
+           for (unsigned int q=0; q<n_q_points; ++q)
+               {
+                 for (unsigned int i=0; i<dofs_per_cell; ++i)
+                	 {
+                       for (unsigned int k=0; k<dofs_per_cell_rt; ++k)
+                           {
+                 	          const double phi_k_u_div = fe_values_rt[velocities].divergence(k,q);
+                 	          cell_matrix_F(i,k) -= (fe_values[interior].value(i,q) *
+                 	                                 phi_k_u_div *
+                 	                                 fe_values.JxW (q));
+                 	        }
+                 	  }
+                 }
+
+            for (unsigned int face_n=0; face_n<GeometryInfo<dim>::faces_per_cell; ++face_n)
+                {
+                   fe_face_values.reinit (cell,face_n);
+                   fe_face_values_rt.reinit (cell_rt,face_n);
+                   for (unsigned int q=0; q<n_face_q_points; ++q)
+                       {
+                     	 const Tensor<1,dim> normal = fe_face_values.normal_vector(q);
+                         for (unsigned int i=0; i<dofs_per_cell; ++i)
+                         	  {
+                         	    for (unsigned int k=0; k<dofs_per_cell_rt; ++k)
+                         	        {
+                         	          const Tensor<1,dim> phi_k_u = fe_face_values_rt[velocities].value (k, q);
+                         	          cell_matrix_F(i,k) += (fe_face_values[face].value(i,q) *
+                         	                                (phi_k_u * normal)*
+                         	                                 fe_face_values.JxW (q));
+                         	         }
+                         	   }
+                         }
+                  }
+
+             cell_matrix_F.mmult(cell_matrix_C,cell_matrix_rt);
+
+             // From previous calculations, we have all the coefficients. Then we get the coefficient @p beta.
+             Vector<double> beta (dofs_per_cell_rt);
+             beta = 0;
+             for(unsigned int k = 0; k<dofs_per_cell_rt;++k)
+                 {
+                   for (unsigned int j = 0; j<dofs_per_cell_rt; ++j)
+                        {
+                          for (unsigned int i = 0; i<dofs_per_cell;++i)
+                              {
+                                 beta(k) += -(cell_solution(i)*
+                                    		  cell_matrix_C(i,j) *
+                                    	      cell_matrix_D(k,j));
+                               }
+                         }
+                  }
+
+              // We calculate numerical velocity on each quadrature point,
+              // find the squares of difference between numerical and exact velocity on the quadrature point.
+              // Then we approximate the error over all quadrature points and add it to the global $L_2$ error.
```

Comment:
`Now, we can calculate the numerical velocity at each quadrature point and compute the $L_2$ error on each cell.`